src/sbl_filing_api/routers/filing.py

```diff
@@ -235,7 +235,7 @@ async def get_latest_submission_report(request: Request, lei: str, period_code:
         file_data = await submission_processor.get_from_storage(
             period_code, lei, str(latest_sub.id) + submission_processor.REPORT_QUALIFIER
         )
-        return FileResponse(path=file_data, media_type="text/csv")
+        return FileResponse(path=file_data, media_type="text/csv", filename=f"{latest_sub.id}validation_report.csv")
```

Comment:
might be easier to read if we put a `_` between the id and the suffix? e.g. `1_validation_report.csv`